python/tests/website/test_web_interface.py

```diff
@@ -0,0 +1,141 @@
+import pytest
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.ui import WebDriverWait
+
+
+@pytest.fixture
+def driver():
```

Comment:
Can we migrate this driver which is used across many test modules into the web_test_utils.py module?